client/me/notification-settings/push-notification-settings/index.jsx

```diff
@@ -769,7 +769,7 @@ class PushNotificationSettings extends React.Component {
 						className="notification-settings-push-notification-settings__settings-icon"
 						icon="bell"
 					/>
-					{ this.props.translate( 'Browser Notifications' ) }
+					{ this.props.translate( 'Browser notifications' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.